deployment-change.md

```diff
@@ -0,0 +1,123 @@
+### Deploying a change to production code
+
+The following interactions make a change happen in the production environment. A narrative explanation follows the diagram.
+
```

Comment:
Can you add the preview image here?

Reply:
![image](https://user-images.githubusercontent.com/75858167/103218563-4f4da300-48e9-11eb-9357-f43f46cd8729.png)